pkg/project/project_test.go

```diff
@@ -52,7 +62,10 @@ func TestDelete(t *testing.T) {
 			if !tt.wantErr == (err != nil) {
 				t.Errorf("project Delete() unexpected error %v, wantErr %v", err, tt.wantErr)
 			}
+
+			if tt.wantMockSetCurrentProjectInvoked != mockSetCurrentProjectInvoked {
+				t.Errorf("Expected invocation of SerCurrentProject: %v but actually invoked: %v. Current Project: %v\n", tt.wantMockSetCurrentProjectInvoked, mockSetCurrentProjectInvoked, client.GetCurrentProjectName())
+			}
```

Comment:
Both values are set by us.  This check makes no sense.

Reply:
I agree both values are set by us but I view it as:
The logic to set(mockSetCurrentProjectInvoked) is the logic expected and the tt.wantMockSetCurrentProjectInvoked is the expected value...
And infact it helped me fix a problem in the code :)